### ISSUE TITLE
StringIndexOutOfBoundsException in BroswerActivity

### ISSUE THREAD

**Issue by codeteo** [kaltura/player-sdk-native-android#273]
You get this error when you try to use sharing menu in player.
I have tried many devices and I get this error in each one of them.

Kaltura player sdk Version : 2.5.17

Error Exception : 

```
09-22 10:15:13.895 1813-1813/MY_PACKAGE.debug E/AndroidRuntime: FATAL EXCEPTION: main
		      Process: MY_PACKAGE.debug, PID: 1813
		      java.lang.RuntimeException: Unable to start activity ComponentInfo{MY_PACKAGE.debug/com.kaltura.playersdk.BrowserActivity}: java.lang.StringIndexOutOfBoundsException: length=0; index=0
				  at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2416)
				  at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
				  at android.app.ActivityThread.-wrap11(ActivityThread.java)
				  at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
				  at android.os.Handler.dispatchMessage(Handler.java:102)
				  at android.os.Looper.loop(Looper.java:148)
				  at android.app.ActivityThread.main(ActivityThread.java:5417)
				  at java.lang.reflect.Method.invoke(Native Method)
				  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
				  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
			   Caused by: java.lang.StringIndexOutOfBoundsException: length=0; index=0
				  at java.lang.String.charAt(Native Method)
				  at android.graphics.Color.parseColor(Color.java:219)
				  at com.kaltura.playersdk.BrowserActivity.onCreate(BrowserActivity.java:31)
				  at android.app.Activity.performCreate(Activity.java:6251)
				  at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107)
				  at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369)
				  at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476) 
				  at android.app.ActivityThread.-wrap11(ActivityThread.java) 
				  at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344) 
				  at android.os.Handler.dispatchMessage(Handler.java:102) 
				  at android.os.Looper.loop(Looper.java:148) 
				  at android.app.ActivityThread.main(ActivityThread.java:5417) 
				  at java.lang.reflect.Method.invoke(Native Method) 
				  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
				  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 

```

**Comment by kaltura-hooks** [comment]
Hi @codeteo,

Thank for you reporting an issue and helping to improve Kaltura!

If you're having some issues getting your player set up, there are quite a few steps that can be taken to figure things out. 


In order to expedite the reporting process, please follow instructions in the doc below, prior to submitting a bug/issue

https://github.com/kaltura/player-sdk-native-android/wiki/How-To-Report-an-Issue-with-the-Kaltura-Android-Player-SDK



If you only have a general question rather than a bug report, please close this issue, post at:
https://forum.kaltura.org/c/community-forums/player-sdk

And add a comment to the Github issue linking to your post.

Thank you in advance!

**Comment by giladna** [comment]
I will check

**Comment by giladna** [comment]
We have inserted a fix to develop branch
will be merged to next version 2.5.30 

https://github.com/kaltura/player-sdk-native-android/pull/278

Please update if it is working for you
